Sources/Turf/GeoJSON.swift

```diff
@@ -0,0 +1,358 @@
+import Foundation
+#if !os(Linux)
+import CoreLocation
+#endif
+
+
+public class BaseFeature: Codable {
```

Comment:
Purely from the perspective of RFC&nbsp;7946, this would be a protocol adopted by both Feature and FeatureCollection, since the real base class of Geometry, Feature, and FeatureCollection would be GeoJSONObject.